pkg/ops/disks.go

```diff
@@ -117,74 +117,6 @@ func GenArmDisk(src, dst string, do schema.Config) func(ctx context.Context) err
 	}
 }
 
-func GenBIOSRawDisk(config schema.Config, srcISO, dst string) func(ctx context.Context) error {
```

Comment:
Isn't this still used here? https://github.com/kairos-io/AuroraBoot/pull/164/files#diff-a40f78b4b8ba4fe8dfa30d4b7ed0b53546de415d98dba2dbcf5ad84099e3de07R131

Reply:
no, it was substituted for a new op with the same name

Reply:
oh, `GenBiosRawDisk` vs `GenBIOSRawDisk` :D . I was sure I was looking at the same string.